cnb/aspera_sync.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+REMOTE_HOST="fileshare.rediris.es"
+REMOTE_USER="adrian@eyeseetea.com"
+ROOT_LOCAL_FOLDER="$HOME/data"
+ROOT_REMOTE_FOLDER='/CNB/CNBPROYECTO1/CBNSHARE'
+
+syncPath(){
+        REMOTE_FOLDER=$1
+        LOCAL_DATA=$2
+        #Get ls with formatted dates and ordered by time
+        lslist=`TZ=utc ls $LOCAL_DATA -altr --color=auto --time-style=full-iso`
+
+        #Get filenames column
+        localfilenames=`echo "$lslist" | awk '{print $9}'`
+
+        #Get aspera folder info and remove all less files table
+        result=`aspera shares browse -i -u $REMOTE_USER --host $REMOTE_HOST --path=$REMOTE_FOLDER  --sort mtime_a | sed -n  '/----/,/----/p' | sed '$d;1d;2d' `
+
+        echo "$result" |
+        while IFS= read -r asperarow;do
```

Comment:
Not sure about this. We check the root files in source (aspera share) and destination (hard disk), but just the root elements (files and directories). What about nested files within directories? This is related with a note below regarding the ability to skip files of aspera.